Dockerfile

```diff
@@ -81,6 +81,7 @@ ARG DFX_METADATA=
 
 RUN touch src/internet_identity/src/lib.rs
 RUN touch src/internet_identity_interface/src/lib.rs
+RUN touch src/canister_sig_util/src/lib.rs
```

Comment:
I think you need to rebase / merge main. This clashes with the fix I made in #1989.

Reply:
Done.